### ISSUE TITLE
driver.rb feedback

### ISSUE THREAD

**Issue by StevenThiesfeld** [ioscowboi/rockpaperscissors#1]
A very minor change that I think would improve presentation would be to display the computer's move after the user enters his/her move.  The best place I see to insert it would be after player 2's move has been validated, after line 69.
`puts "Computer played #{player1_move_create}"` would be all that's needed.

**Comment by ioscowboi** [comment]
Yes, I had thought of that but then I forgot to actually do it. It appears it's good to have a second set of eyes.
